src/renderer/components/table/table.tsx

```diff
@@ -24,11 +25,9 @@ export interface TableProps extends React.DOMAttributes<HTMLDivElement> {
   selectable?: boolean; // Highlight rows on hover
   scrollable?: boolean; // Use scrollbar if content is bigger than parent's height
   storageKey?: string;  // Keep some data in localStorage & restore on page reload, e.g sorting params
-  sortable?: {
-    // Define sortable callbacks for every column in <TableHead><TableCell sortBy="someCol"><TableHead>
-    // @sortItem argument in the callback is an object, provided in <TableRow sortItem={someColDataItem}/>
-    [sortBy: string]: TableSortCallback;
-  };
+  // Define sortable callbacks for every column in <TableHead><TableCell sortBy="someCol"><TableHead>
+  // @sortItem argument in the callback is an object, provided in <TableRow sortItem={someColDataItem}/>
```

Comment:
Multi line comments, especially with (js/ts)doc formatting should be done with a multi line comment.
```suggestion
  /**
   * Define sortable callbacks for every column in <TableHead><TableCell sortBy="someCol"><TableHead>
   * @sortItem argument in the callback is an object, provided in <TableRow sortItem={someColDataItem}/>
   */
```

Reply:
Fixed